fe/fe-core/src/main/java/org/apache/doris/catalog/SchemaTable.java

```diff
@@ -281,6 +281,23 @@ public static Builder builder() {
                                                  .column("XA", ScalarType.createVarchar(3))
                                                  .column("SAVEPOINTS", ScalarType.createVarchar(3))
                                                  .build()))
+                    .put("views",
+                        new SchemaTable(
+                                SystemIdGenerator.getNextId(),
+                                "views",
+                                TableType.SCHEMA,
+                                builder()
+                                        .column("TABLE_CATALOG", ScalarType.createVarchar(512))
+                                        .column("TABLE_SCHEMA", ScalarType.createVarchar(64))
+                                        .column("TABLE_NAME", ScalarType.createVarchar(64))
+                                        .column("VIEW_DEFINITION", ScalarType.createVarchar(8096))
+                                        .column("CHECK_OPTION", ScalarType.createVarchar(8))
+                                        .column("IS_UPDATABLE", ScalarType.createVarchar(3))
+                                        .column("DEFINER", ScalarType.createVarchar(77))
+                                        .column("SECURITY_TYPE", ScalarType.createVarchar(7))
```

Comment:
why this column length is 77, you should add a comment, this column in mysql is 288

Reply:
Maybe you get another version of mysql. the sql I got is like this.